developerportal/templates/organisms/filter-list.html

```diff
@@ -29,6 +29,12 @@ <h2>{{ heading }}</h2>
               {% for resource in resources %}
                 {% if type == "article" %}
                   {% include "molecules/cards/card.html" with resource=resource.article filter_target=True show_author=True %}
+                {% elif type == "article_or_video" %}
+                  {% if resource.video %}
+                    {% include "molecules/cards/card.html" with resource=resource.video filter_target=True %}
+                  {% else %}
+                    {% include "molecules/cards/card.html" with resource=resource.article filter_target=True show_author=True %}
+                  {% endif %}
```

Comment:
Again, not the super-nicest approach, but hopefully explicit enough to avoid surprises when we read this back in the future